descriptors/ModuleDescriptor-template.json

```diff
@@ -1450,6 +1459,11 @@
         "acquisitions-units.units.item.delete"
       ]
     },
+    {
+      "permissionName": "orders.bypass-acquisitions-units",
```

Comment:
orders.bypass-**acquinisions-units** is more correct in terms of grammar? in all places we use just acquision-units
![image](https://github.com/folio-org/mod-orders/assets/25097693/92ee3415-bb8e-4a42-91c0-8b612f9510a1)

Reply:
I will change it to "orders.bypass-acquisition-units".